integration-tests/example-project/src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
+quarkus.openapi-generator.codegen.validateSpec=false
```

Comment:
Please add another example project to the IT module. This project is already saturated with use cases.

Reply:
done